@here/harp-materials/package.json

```diff
@@ -20,6 +20,7 @@
     },
     "license": "Apache-2.0",
     "dependencies": {
+        "@here/harp-datasource-protocol": "^0.10.0",
```

Comment:
Do we really need this dependeny? I would rather add the mapping from cap type to define to datasource protocoll.

Reply:
Could you please clarify what did you exactly mean? How it should be mapped?

Reply:
As far as I understood you added this dependency so that you can auto-generate the mapping from cap type (defined in datasource-protocol) to DEFINE name. I would rather do that mapping by hand (i.e a switch statement) than adding this dependency

Reply:
Yes, I'm trying to create one source of truth, and I also want TypeScript to guarantee me in future that if there are changes in this code, they will be accurately processed in all the necessary places. I think an approach like this can help write more error resistant code. I can change it of course but if you agree I would like to live it as is.